benchbuild/__init__.py

```diff
@@ -1,13 +1,39 @@
 """
-Setup plugins.
+Public API of benchbuild.
 """
+import sys
+from typing import Callable, Optional
+from plumbum.commands import BaseCommand
+from plumbum import local, Path
+
 from . import experiments as __EXPERIMENTS__
 from . import projects as __PROJECTS__
 from . import reports as __REPORTS__
-from .settings import CFG as __CFG__
+from .downloads.base import BaseSource
+from .project import Project
+from .settings import CFG
+from .utils import compiler, download, run
 from .utils import settings as __SETTINGS__
+from .utils import wrapping
 
 __EXPERIMENTS__.discover()
 __PROJECTS__.discover()
 __REPORTS__.discover()
-__SETTINGS__.setup_config(__CFG__)
+__SETTINGS__.setup_config(CFG)
+
+cwd = local.cwd
+env = local.env
+path = local.path
+
+wrap = wrapping.wrap
+watch = run.watch
+
+del local
+del sys
+del BaseCommand
+del BaseSource
+del Path
+del __EXPERIMENTS__
+del __PROJECTS__
+del __REPORTS__
+del __SETTINGS__
```

Comment:
Do you want to delete all imports here or just some because the list is incomplete.

Reply:
No not all Imports.

Some have been required to initialize Experiments/Projects/Reports  and Settings properly. However, I noticed some weird ones. I'll double check tomorrow.

Reply:
No, just the imports that do not belong to the public API but were required to initialize everything properly.
I just noticed, the list is super weird. I'll double check tomorrow.